bin/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "enable": true,
+  "debug": true,
+  "providers": {
+    "file:///Users/hiteshsagtani/dev/openctx/provider/slack-channel-context/dist/bundle.js": true
```

Comment:
Accidental git add?

Reply:
oh damn, yes pushed it by mistake, thanks for noticing !!